PoseEuler pose
---
uint8 location_id
